src/sentry/audit_log/register.py

```diff
@@ -207,9 +207,10 @@
 default_manager.add(events.IntegrationAddAuditLogEvent())
 default_manager.add(events.IntegrationEditAuditLogEvent())
 default_manager.add(events.IntegrationRemoveAuditLogEvent())
+default_manager.add(events.IntegrationDisabledAuditLogEvent())
 default_manager.add(
     AuditLogEvent(
-        event_id=113,
+        event_id=114,
```

Comment:
Why is the event id for adding sentry apps changing?

Reply:
This would cause inconsistent data for adding sentry apps - I can see that you ran out of numbers for integration audit log events, I think instead of changing one you should just increment by a lot like do 200.

Reply:
The integration events were 109-112 and Sentry App was 113 and 115-117 so I changed the Sentry App 113 to 114 to accommodate the new Integration disable event at 113. I can make our new event id 108 instead, so Integration events will be 108-112 and I won't have to change the Sentry App event ids.